### DISCUSSION TITLE
How to show/hide a navigation entry depending on the panel?

### DISCUSSION THREAD

**Discussion by micraux** [filamentphp/filament#7966] [Help] [answered]
I would like to hide the "Teams" entry in the "App" panel but still want it to be present in the "Admin" panel.

I've read the "conditionally-hiding-navigation-items" section in the documentation, but this doesn't suit my needs as I want to conditionally show/hide the entry depending on which panel I am, not on a role/permission.

What would be the best way to achieve that?

**Comment by danharrin** [comment]
Do not register the resource for that panel in the first place. By default, panels have different directories they register resources from.

**Comment by micraux** [comment]
Oops... :-)

The problem is I already had the "Team" resource in my "App" panel, so when I created my "Admin" panel all the resources have become available in both panels. I just tried creating a new resource, and I can see the choice "_Which panel would you like to create this in?_" so I understand your answer now :-)

But... I created a new Client resource (in the "App" panel) and it is showing in both panels :-(

I suppose I shouldn't see it in my "Admin" menu panel, is it correct?

**Reply by micraux** [reply]
I guess I messed up things renaming panels...
You wrote: "_By default, panels have different directories they register resources from_."
Can you tell me where I can find these files/directories?

**Reply by danharrin** [reply, marked as answer]
You see how this panel discovers resources in App/Filament/App/Resources instead of App/Filament/Admin/Resources or App/Filament/Resources

<img width="970" alt="Screenshot 2023-08-22 at 11 52 35" src="https://github.com/filamentphp/filament/assets/41773797/59fa098c-45ae-4200-b7ad-b52997c70102">

**Comment by micraux** [comment]
Nice, I was suspecting that could be the solution: in fact, because I messed a little bit, the Filament subdirectories were not created, so all my resources were in the Filament directory!

Thank a lot!
